hal-base/hal/constants.py

```diff
@@ -7,12 +8,12 @@
 #############################################################################
 
 
-class RuntimeType:
+class RuntimeType(enum.IntEnum):
     Athena = 0
     Mock = 1
 
 
-class UsageReporting:
+class UsageReporting(enum.IntEnum):
```

Comment:
Given that this has a lot of duplicates with different semantics, I don't think this makes sense as a single enum.

Reply:
We probably don't need to. I don't think anything is type hinted as UsageReporting

Reply:
Yeah, this just causes problems:

```
>>> hal.UsageReporting.kLanguage_LabVIEW
<UsageReporting.kResourceType_Module: 1>
```